lib/screens/v2/widget_instance/elevator_status.ex

```diff
@@ -182,16 +188,22 @@ defmodule Screens.V2.WidgetInstance.ElevatorStatus do
          %Alert{effect: :elevator_closure, informed_entities: entities} = alert,
          %__MODULE__{now: now, stop_sequences: stop_sequences} = t
        ) do
-    stations = get_stations_from_entities(entities)
+    informed_platforms =
+      for %{stop: stop} when is_binary(stop) <- entities,
+          match?({_n, ""}, Integer.parse(stop)),
+          do: stop
+
     parent_station_id = parent_station_id(t)
+    platform_stop_ids = platform_stop_ids(t)
 
     flat_stop_sequences =
       stop_sequences
       |> List.flatten()
 
     Alert.happening_now?(alert, now) and
-      Enum.any?(stations, fn station ->
-        station in flat_stop_sequences and station != parent_station_id
+      Enum.any?(informed_platforms, fn platform ->
+        platform != parent_station_id and platform not in platform_stop_ids and
+          platform in flat_stop_sequences
```

Comment:
Trying to understand what `platform_stop_ids` is in the elevator status config, and why it is needed on pre-fare screens.  (I do understand why we'd need the ids for the elevator screens down the road)

That will help me understand what this piece is doing:  `platform not in platform_stop_ids`

Reply:
The reason we need them is to know what `platform_ids` are located at the `parent_station`. With these elevator alerts, the `informed_entities` list contains (most importantly) the parent station. So an elevator closure at Sullivan Square will have an entry for `place-sull` in the `informed_entities` list. There will also be an entry for each platform at Sullivan Square. These are numerical ids that do not tell us anything by looking at them. We need to know these ids when looking for alerts on connecting routes. If we do not exclude `platform_ids` at the home station, `active_at_home` alerts will show in the `active_on_connecting_routes` list. So if an alert affects a stop in the `stop_sequences` list and does not have an entry for any associated ids for the parent station, we know it is on the connecting route and not for the station itself.

Reply:
Do we need to have `platform_stop_ids` be a configuration, though?  Is there any way to get the `platform_stop_ids` from the API given the parent station so that we don't have to hardcode the `parent_station_id` _and_ the `platform_stop_ids` for each screen?

Reply:
I don't remember the exact reason we went this way originally. I think it was to do with how complicated the query was getting. We went to Paul and asked how he felt about it being in the config and he felt that was reasonable. Mostly because these IDs are not going to be changing.

Reply:
Mmm, I guess thats true, ids won't be changing.  Just out of curiosity, I'll ask it in slack.  But otherwise, you've got my approval 👍